### DISCUSSION TITLE
feroxbuster out of date since 2 years, weird commits ref

### DISCUSSION THREAD

**Discussion by pierrelegall** [BlackArch/blackarch#4162] [Q&A]
Hello Blackarchers!

I see the `feroxbuster` package last version in the BlackArch repository is `2.7.1.r11.g53e3420-1`.

```
❯ pacman -Ss feroxbuster
blackarch/feroxbuster 2.7.1.r11.g53e3420-1 (blackarch blackarch-webapp blackarch-scanner blackarch-fuzzer)
    A fast, simple, recursive content discovery tool written in Rust.
```

Two questions:

- **Is it reasonable for `feroxbuster` to be out dated since 2 years?**

```
~/Code/feroxbuster tags/2.7.1
❯ git log -n 1 --date=short
commit f51993c (HEAD, tag: 2.7.1)
Merge: 4926651 9093ffb
Author: epi <43392618+epi052@users.noreply.github.com>
Date:   2022-05-12

    Merge pull request #575 from epi052/all-contributors/add-postmodern
    
    docs: add postmodern as a contributor for ideas
```

- **I can't found the `g53e3420` commit on the official feroxbuster repository, why?**

```
~/Code/feroxbuster tags/2.7.1
❯ git log g53e3420
fatal: ambiguous argument 'g53e3420': unknown revision or path not in the working tree.
```

**Comment by ikstream** [comment]
The commit ref follows the [guidelines with version tags](https://wiki.archlinux.org/title/VCS_package_guidelines). The actual git reference is `53e3420`. 

Nevertheless you are right, the `feroxbuster` package seems to have a build issue that have not been reported so far. 
The PKGBUILD is on `2.10.4` but it hasn't been build since August 2022. 

Thanks for reporting

**Reply by pierrelegall** [reply]
> The actual git reference is 53e3420.

Oups, I copy-pasted the disturbing "g" in from of the commit ref ;p

**Comment by noraj** [comment]
Since it's definitely an issue, let's continue there: https://github.com/BlackArch/blackarch/issues/4163
